frontend/lib/src/components/elements/Metric/Metric.test.tsx

```diff
@@ -68,25 +69,26 @@ describe("Metric element", () => {
         value: LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED,
       },
     })
-    const wrapper = mount(<Metric {...props} />)
-    expect(wrapper.find("StyledMetricLabelText").prop("visibility")).toEqual(
-      LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED
+    render(<Metric {...props} />)
+    expect(screen.getByTestId("stMetricLabel")).toHaveAttribute(
+      "visibility",
+      String(LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED)
     )
   })
 
   it("renders direction icon based on props - red/up", () => {
     const props = getProps()
-    const wrapper = mount(<Metric {...props} />)
-    expect(wrapper.find("StyledMetricDeltaText").find("svg")).toBeDefined()
+    render(<Metric {...props} />)
+    expect(screen.getByTestId("stMetricDeltaIcon")).toBeInTheDocument()
   })
 
   it("renders direction icon based on props - green/down", () => {
     const props = getProps({
       color: MetricProto.MetricColor.GREEN,
       direction: MetricProto.MetricDirection.DOWN,
     })
-    const wrapper = mount(<Metric {...props} />)
-    expect(wrapper.find("StyledMetricDeltaText").find("svg")).toBeDefined()
+    render(<Metric {...props} />)
+    expect(screen.getByTestId("stMetricDeltaIcon")).toBeInTheDocument()
```

Comment:
Same here, after testing icon renders, check the style color - green should be `"#09ab3b"`

Reply:
Fixed!